### PR TITLE
Add lockhunter. Closes #1301.

### LINKED ISSUE #1301: [Request] LockHunter
Currently unlocker can be installed via scoop but at least on my computer it hasn't been really working correctly. LockHunter is a much better application and has never let me down so far.

### PR DESCRIPTION
**Pull request by matthewjberger**
Lockhunter is a file unlocker :smile:

### CODE REVIEW

File: lockhunter.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "homepage": "https://lockhunter.com/index.htm",
+    "version": "3.2.3",
+    "description": "a foolproof file unlocker",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://lockhunter.com/exe/lockhuntersetup_3-2-3.exe",
+            "hash": "59e7051cb079be49ac2bb4dd44cbdb6d4e11b6466d779465c7d1a3ca59272663",
+            "pre_install": [
```
Comment (Ash258) [inline lockhunter.json]: Make that `pre_install` non architecture specific, since its same

Comment (Ash258) [reply lockhunter.json]: Or whats difference between `LockHunter,1` and `LockHunter,2`?

Comment (matthewjberger) [reply lockhunter.json]: Same as the 32-Bit version? Innounpack unpacks files named `LockHunter,1.exe` which is 64-bit and `LockHunter,2.exe` which is 32-Bit. The architecture specific `pre_install` renames the appropriate file and deletes the other.

Comment (Ash258) [reply lockhunter.json]: URL and hashes are same too. Make it only something like this:

```json
"url": "URL",
"hash": "Hash",
"architecture" : {
    "xbi": {
         "pre_install": "Rename-Item \"$dir\\LockHunter,X.exe\" \"$dir\\lockhunter.exe\""
    }
},
"post_install": "Remove-Item \"$dir\\LockHunter*,*.exe\""
```

File: lockhunter.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                "rename-item \"$dir\\LockHunter,2.exe\" \"$dir\\lockhunter.exe\"",
+                "remove-item \"$dir\\LockHunter*,*.exe\""
+            ]
+        }
+    },
+    "innosetup": true,
+    "checkver": {
+        "url": "https://lockhunter.com/download.htm",
+        "re": "Version: (\\d+\\.\\d+\\.\\d+)"
```
Comment (Ash258) [inline lockhunter.json]: Change to `Version: ([\\d\\.]+);`

File: lockhunter.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
+            },
+            "32bit": {
+                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
+            }
+        }
+    },
+    "bin": [
```
Comment (Ash258) [inline lockhunter.json]: Change to `"bin": "lockhunter.exe"`

File: lockhunter.json (hunk 1 above)
Comment (matthewjberger) [reply lockhunter.json]: See the previous message... The two non-service exe files unpacked by `innoup` are of different architectures

Comment (Ash258) [reply lockhunter.json]: NO!

```json
{
    "homepage": "https://lockhunter.com/index.htm",
    "version": "3.2.3",
    "description": "a foolproof file unlocker",
    "license": "Freeware",
    "url": "https://lockhunter.com/exe/lockhuntersetup_3-2-3.exe",
    "hash": "59e7051cb079be49ac2bb4dd44cbdb6d4e11b6466d779465c7d1a3ca59272663",
    "architecture": {
        "64bit": {
            "pre_install": "Rename-Item \"$dir\\LockHunter,1.exe\" \"$dir\\lockhunter.exe\""
        },
        "32bit": {
            "pre_install": "Rename-Item \"$dir\\LockHunter,2.exe\" \"$dir\\lockhunter.exe\""
        }
    },
    "post_install": "Remove-Item \"$dir\\LockHunter*,*.exe\"",
    "innosetup": true,
    "checkver": {
        "url": "https://lockhunter.com/download.htm",
        "re": "Version: ([\\d\\.]+)"
    },
    "bin": "lockhunter.exe",
    "shortcuts": [
        [
            "lockhunter.exe",
            "LockHunter"
        ]
    ],
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
            },
            "32bit": {
                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
            }
        }
    },
}

```

**Pull request merged**